skins/moono-lexicon/dialog.css

```diff
@@ -277,19 +274,17 @@ a.cke_dialog_tab_disabled {
 
 /* The close button at the top of the dialog. */
 a.cke_dialog_close_button {
-	background-image: url(images/close.png);
-	background-repeat: no-repeat;
+	background-image: url("data:image/svg+xml;charset=utf8,%0A%3Csvg%20fill='%236b6c7e'%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20512%20512'%3E%3Cpath%20class='lexicon-icon-outline'%20d='M301.1,256.1l137.2-137.2c30.1-30.1-16.8-73.6-45.2-45.2L255.9,210.8L118.6,73.7C88.6,43.6,45,90.5,73.4,118.9l137.2,137.2L73.3,393.3c-28.9,28.9,15.8,74.6,45.2,45.2l137.3-137.2l137.3,137.2c28.9,28.9,74.2-16.3,45.2-45.2L301.1,256.1z'/%3E%3C/svg%3E%0A") !important;	position: absolute;
```

Comment:
it is what it is... but `position: absolute` should maybe be in a different line?